README.md

```diff
@@ -0,0 +1,357 @@
+
+All the files for this example can be found in the `/docs/example1` folder.
+
+In order to setup the initial scenario, the RabbitMQ users must be created in the broker. Go to the `bin/` folder of
+your RabbitMQ instance and execute the `setupTest.sh` script. This script will create three users, on for each role in
+our example app: journalist, audience and editor. It also gives permissions for accessing all kind of entities in the
+default VHost. Take into account that this permissions are used at a different level than the routing key permissions
+offered by the plugin: this plugin does not override default RabbitMQ security, it adds a new level of security that
+takes into account not only RabbitMQ resources, but also routing keys.
+
+Once the users have been created in the broker, it's time to load the ACL file. There is an example ACL file called
+`newsacl` in the `/docs/example1` folder, with the permissions described in the problem description.
+
+The file can be loaded using the RabbitMQ Erlang console. If you are testing the plugin using the `make run-broker`
+build task, show the Erlang console by pressing Enter. If you are using a standalone RabbitMQ, you can issue Erlang
+commands to the server through the `rabbitmqctl eval <expression>` command. In the following, we will use the latter
+method (in order to use the former, just issue the expressions directly through the Erlang console).
+
+Use the following command to show the current ACL DB contents:
+
+```
+./rabbitmqctl eval "aclstore:list_permissions()."
+```
+
+If its the first time you execute this plugin, you should see an empty array. Now, load the ACL file by using the
+following command:
+
+```
+./rabbitmqctl eval "aclstore:load_permissions_file(<absolute_path_to_the_file>)."
+```
+
+Executing the permission list command again should give the following result:
+
+```
+[
+{"audience",read,"news/categories/*"}
+,{"editor",read,"news/categories/*"}
+,{"editor",readwrite,"news/events`"}
+,{"journalist",readwrite,"news/categories/*"}
+,{"journalist",read,"news/events`"}
+]
+```
+
+Now RabbitMQ is configured to start exchanging messages between our roles. In order to test the message interchange,
+the RabbitMQ ACL Tool will be used.
+
+In order to simulate the behavior depicted in the problem description, open two consoles and use the ACL Tool to
+listen for news in `news/categories/sports` with journalist permissions in one tab and with audience permissions in
+the other.
+
+```
+./acltool.js listen -U journalist -P password newsfeed "news/categories/sports"
+./acltool.js listen -U audience -P password newsfeed "news/categories/sports"
+```
+
+Now, if we try publishing a piece of news using the journalist user:
+```
+./acltool.js publish -U journalist -P password newsfeed "news/categories/sports" "New kind of seven-winged chicken developed"
+```
+
+We should see the news appearing in both listeners. If we now try to publish with the audience user, we should get an error:
```

Comment:
The command below this line is not about publishing but "listen". Maybe is there a typo here?